### PR TITLE
Awais786/ecom 5802 publisher model changes

### PR DESCRIPTION
**Pull request by awais786**
Publisher model changes.
**Course Model**
- team_admin
- Image

**CourseRun Model**
- is_seo_review
- video_language

### CODE REVIEW

**Comment by awais786** [comment]
@waheedahmed @bderusha Please review this PR.
@bderusha for course-> number field i have created the separated sub-task.
https://openedx.atlassian.net/browse/ECOM-5838
I will merge that migration with minimum changes.

File: course_discovery/apps/publisher/models.py [hunk 1]
```diff
@@ -177,9 +192,8 @@ class CourseRun(TimeStampedModel, ChangedByMixin):
     is_micromasters = models.BooleanField(default=False)
     micromasters_name = models.CharField(max_length=255, null=True, blank=True)
     contacted_partner_manager = models.BooleanField(default=False)
-    seo_review = models.TextField(
-        default=None, null=True, blank=True, help_text=_("SEO review on your course title and short description")
-    )
+    seo_review = models.BooleanField(default=False)
```
Comment (waheedahmed) [inline course_discovery/apps/publisher/models.py]: Change it to `is_seo_review`.

**Review by waheedahmed — APPROVED** [review]
One comment, 👍 once addressed.

**Comment by awais786** [comment]
@bderusha Please review.

**Comment by awais786** [comment]
@bderusha Please review this PR.

File: course_discovery/apps/publisher/models.py [hunk 2]
```diff
@@ -112,6 +114,19 @@ class Course(TimeStampedModel, ChangedByMixin):
...
+    team_admin = models.ForeignKey(User, null=True, blank=True, related_name='team_admin_user')
+    image = StdImageField(
+        upload_to=UploadToFieldNamePath(
+            populate_from='number',
+            path='media/publisher/courses/images'
+        ),
+        blank=True,
+        null=True,
+        variations={
+            'large': (2120, 1192),
```
Comment (bderusha) [inline course_discovery/apps/publisher/models.py:126]: Do we know what sizes this file should be?  Can we find out?  This seems massive.

File: course_discovery/apps/publisher/models.py [hunk 3]
```diff
@@ -10,10 +10,12 @@
 from guardian.shortcuts import assign_perm
 from simple_history.models import HistoricalRecords
 from sortedm2m.fields import SortedManyToManyField
+from stdimage.models import StdImageField
 
 from course_discovery.apps.core.models import User, Currency
 from course_discovery.apps.course_metadata.choices import CourseRunPacing
 from course_discovery.apps.course_metadata.models import LevelType, Subject, Person, Organization
+from course_discovery.apps.course_metadata.utils import UploadToFieldNamePath
```
Comment (bderusha) [inline course_discovery/apps/publisher/models.py:18]: If we need this utility as well we should move it out of course_metadata and into core

**Review by bderusha — APPROVED** [review]
👍 Once comments addressed

File: course_discovery/apps/publisher/models.py (hunk 2 above)
Comment (awais786) [reply course_discovery/apps/publisher/models.py:126]: @bderusha in mockup Image have this **Size: 2120 x 1192 pixels**
I think it should be like 

```
variations={
            'large': (1440, 480),
            'medium': (726, 242),
            'small': (435, 145),
            'x-small': (348, 116),
        }
```

File: course_discovery/apps/publisher/models.py (hunk 3 above)
Comment (awais786) [reply course_discovery/apps/publisher/models.py:18]: @bderusha let this change go this way.  This file is in-use in migration here
https://github.com/edx/course-discovery/blob/master/course_discovery/apps/course_metadata/migrations/0020_auto_20160819_1942.py#L24

File: course_discovery/apps/publisher/models.py (hunk 2 above)
Comment (awais786) [reply course_discovery/apps/publisher/models.py:126]: @bderusha we can tag Roderick. Do you know his git user name ?

Comment (roderickmorales) [reply course_discovery/apps/publisher/models.py:126]: @bderusha @awais786 That's the size that we currently ask for in the word document. Maybe Deen can touch base with the PMs to find out why we request such a big image.

Comment (awais786) [reply course_discovery/apps/publisher/models.py:126]: We will update the sizes accordingly in future. For time being it is fine with 

```
variations={
            'large': (2120, 1192),
             'large': (1440, 480),
        }
```

**Pull request merged**